fix(desktop): correct Antares Desktop packaging

laurent-laporte-pro opened the pull request.

laurent-laporte-pro committed:
fix(desktop): improve Antares Web packaging to preserve file execution mode (chmod)

laurent-laporte-pro committed:
fix: upgrade the example study to v8.5

laurent-laporte-pro committed:
fix: correct the Ubuntu packaging to set executable permissions

laurent-laporte-pro committed:
fix: update the GH workflow to preserve symlinks and improve documentation

laurent-laporte-pro committed:
fix: correct the 7z parameters used to archive files

laurent-laporte-pro committed:
fix: add missing files in ZIP (use -r flag) and avoid `dist/package` subdirectory

laurent-laporte-pro committed:
fix: correct the target path of `commit_id` on Ubuntu

laurent-laporte-pro committed:
fix: change the name of the final package to avoid confusion

ghost commented:
### Bug Fixes

*  **desktop:** improve Antares Web packaging to preserve file execution mode (chmod) ([c33e174](https://github.com/AntaresSimulatorTeam/AntaREST/commit/c33e174c472a45aa23dff236aee346de8bcf8bc7))
* upgrade the example study to v8.5 ([2abb521](https://github.com/AntaresSimulatorTeam/AntaREST/commit/2abb521d1897432ef77bbcfc1d50345f5f7e810e))
* correct the Ubuntu packaging to set executable permissions ([6e55cd8](https://github.com/AntaresSimulatorTeam/AntaREST/commit/6e55cd8e24865e4b87c3fcb8e8b6a4172d4b65a8))
* update the GH workflow to preserve symlinks and improve documentation ([6eecc03](https://github.com/AntaresSimulatorTeam/AntaREST/commit/6eecc03ca9da119f1ce8ec74fed8b1fd59074bc2))
* correct the 7z parameters used to archive files ([87cb45f](https://github.com/AntaresSimulatorTeam/AntaREST/commit/87cb45fe4a3f63605fad0732f25898baffd8253d))
* add missing files in ZIP (use -r flag) and avoid `dist/package` subdirectory ([371d88b](https://github.com/AntaresSimulatorTeam/AntaREST/commit/371d88be16c86148883ca04d90e65cec37093bbd))
* correct the target path of `commit_id` on Ubuntu ([d55102a](https://github.com/AntaresSimulatorTeam/AntaREST/commit/d55102abfb90d7c0b026dcae1ea62893c63b775e))
* change the name of the final package to avoid confusion ([131db06](https://github.com/AntaresSimulatorTeam/AntaREST/commit/131db0671afec0294191fe480e5c7452e44d7953))
* correct the executable paths ([a6e3d01](https://github.com/AntaresSimulatorTeam/AntaREST/commit/a6e3d01b959ed8dc1675f6008a3a76ec717c6d48))
* correct the final package name ([a9a717f](https://github.com/AntaresSimulatorTeam/AntaREST/commit/a9a717f864dca861a58d6e45ea36d3b346c5bd0d))


### Code Refactoring

* move the Pyinstaller stages in the `package_antares_web.sh` ([70a2774](https://github.com/AntaresSimulatorTeam/AntaREST/commit/70a2774d32ba04bce639e5efcfb5e4223fc1108f))


### Contributors

<a href="https://github.com/laurent-laporte-pro">laurent-laporte-pro</a>

<details>
<summary>Commit-Lint commands</summary>
<br />

You can trigger Commit-Lint actions by commenting on this PR:
- `@Commit-Lint merge patch` will merge dependabot PR on "patch" versions (X.X.Y - Y change)
- `@Commit-Lint merge minor` will merge dependabot PR on "minor" versions (X.Y.Y - Y change)
- `@Commit-Lint merge major` will merge dependabot PR on "major" versions (Y.Y.Y - Y change)
- `@Commit-Lint merge disable` will desactivate merge dependabot PR
- `@Commit-Lint review` will approve dependabot PR
- `@Commit-Lint stop review` will stop approve dependabot PR



</details>

laurent-laporte-pro committed:
fix: correct the executable paths

laurent-laporte-pro committed:
refactor: move the Pyinstaller stages in the `package_antares_web.sh`

By running Pyinstaller in the Bash script, it is easier to debug and find missing files.

laurent-laporte-pro committed:
fix: correct the final package name
